pinax/blog/models.py

```diff
@@ -61,6 +62,9 @@ class Post(models.Model):
 
     STATE_CHOICES = PINAX_BLOG_STATE_CHOICES
 
+    if settings.PINAX_BLOG_SCOPING_MODEL is not None:
+        scoped_for = models.ForeignKey(settings.PINAX_BLOG_SCOPING_MODEL, related_name="blog_posts")
```

Comment:
`scoped_for` and `scoper` seemed to be used interchangably throughout variable naming for this object. The former is a slightly confusing name. Reading code like `post.scoped_for` feels backwards to me. `post.scoper` would be better IMO.